### DISCUSSION TITLE
Possible to disable frontend completeyl and just use API?

### DISCUSSION THREAD

**Discussion by taylor-shift** [OpenSignLabs/OpenSign#983] [General]
I am thinking of integrating OpenSign with my backend...Is there a way I can run OpenSign in "headless" mode?

**Comment by andrew-opensignlabs** [comment]
You can use OpenSign API to integrate with your existing app - https://docs.opensignlabs.com/docs/API-docs/opensign-api-v-1

API is available on pro, teams & enterprise plan.
